### PR TITLE
update docs with some issues with sriov and general naming

### PR DESCRIPTION
**Pull request by alknopfler**
Changes:
- SRIOV improvements from feedback (vfio-pci and netdevice, upstream references github...)
- Directed vs Direct

### CODE REVIEW

File: asciidoc/product/atip-features.adoc [hunk 1]
```diff
@@ -419,9 +417,9 @@ This part could be done in two ways, using the `CLI` or using the `Rancher UI`.
 Install Operator from CLI::
 +
 ----
-helm repo add rancher-charts suse-edge https://suse-edge.github.io/charts
-helm install sriov-crd suse-edge/sriov-crd
-helm install install sriov-network-operator suse-edge/sriov-network-operator
+helm repo add suse-edge https://suse-edge.github.io/charts
+helm install sriov-crd suse-edge/sriov-crd -n kube-system
```
Comment (e-minguez) [inline asciidoc/product/atip-features.adoc:421]: Is it really needed to use the kube-system namespace?

Comment (alknopfler) [reply asciidoc/product/atip-features.adoc:421]: we can install on a separate namespace (sriov-network or something specific), for sure, but as we've got the cni on kube-system I added to kube-system.

Comment (e-minguez) [reply asciidoc/product/atip-features.adoc:421]: Upstream https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/quickstart.md and OpenShift https://github.com/openshift/sriov-network-operator/blob/master/doc/quickstart.md deploy the operator on the sriov-network-operator namespace. I think we should align to those as well.

Comment (alknopfler) [reply asciidoc/product/atip-features.adoc:421]: ok, makes sense...I will modify with the upstream namespace

Comment (alknopfler) [reply asciidoc/product/atip-features.adoc]: thanks @e-minguez

File: asciidoc/product/atip-features.adoc [hunk 2]
```diff
@@ -419,23 +417,28 @@ This part could be done in two ways, using the `CLI` or using the `Rancher UI`.
 Install Operator from CLI::
 +
 ----
-helm repo add rancher-charts suse-edge https://suse-edge.github.io/charts
-helm install sriov-crd suse-edge/sriov-crd
-helm install install sriov-network-operator suse-edge/sriov-network-operator
+helm repo add suse-edge https://suse-edge.github.io/charts
+helm install sriov-crd suse-edge/sriov-crd -n sriov-network-operator
+helm install install sriov-network-operator suse-edge/sriov-network-operator -n sriov-network-operator
```
Comment (hardys) [inline asciidoc/product/atip-features.adoc:422]: Should we add this chart to the IBS builds so it's consistent with the other charts?

**Review by hardys — APPROVED** [review]
LGTM but we should discuss the plan for the sriov chart

**Pull request merged**